endgame_postprocessing/model_wrappers/sch/README.md

```diff
@@ -0,0 +1,96 @@
+# Usage
+
+Having installed the endgame post processing pipeline, there exists functions for running 
+both the STH and SCH pipelines.
+
+## STH
+
+The STH pipeline can either create a "any worm" result, or a single worm result. 
+
+### Any Worm
+
+```python
+import endgame_postprocessing.model_wrappers.sch.run_sch as run_sch
+
+input_dir = # TODO
+# Collect all of the worm directories that are in the input_dir
+worm_directories = next(os.walk(input_dir))[1]
+run_sch.run_sth_postprocessing_pipeline(
+    input_dir,
+    "desired output directory",
+    worm_directories,
+    1,
+    skip_canonical=False,
+)
+```
+
+### Single Worm
+
+Alternatively, can be run for each worm separately:
+
+```python
+import endgame_postprocessing.model_wrappers.sch.run_sch as run_sch
+
+input_dir = # TODO
+worm_directories = next(os.walk(input_dir))[1]
+for worm_directory in worm_directories:
+    run_sch.run_sth_postprocessing_pipeline(
+        input_dir,
+        f"local_data/sth-output-single-worm/{worm_directory}",
+        [worm_directory],
+        1,
+        skip_canonical=False,
+    )
+```
+
+## Schisto
+
+At the moment, Schisto can only be run on each worm individually: 
+
+```python
+import endgame_postprocessing.model_wrappers.sch.run_sch as run_sch
+
+root_input_dir = # TODO
+worm_directories = next(os.walk(root_input_dir))[1]
+for worm_directory in worm_directories:
+    run_sch.run_sch_postprocessing_pipeline(
+        f"{root_input_dir}/{worm_directory}",
+        f"local_data/sch-output-single-worm/{worm_directory}",
+        skip_canonical=False,
+    )
+```
+
+It expects files inside this directory (arbitrarily nested) matching the following regex:
+
+```
+ntdmc-(?P<iu_id>(?P<country>[A-Z]{3})\d{5})-[\w_]+-group_001-(?P<scenario>scenario_\w+)-survey_type_kk2-group_001-200_simulations.csv
+```
+
+Additionally, each worm directory needs to have a copy of the `PopulationMetadatafile.csv`.
+That is, the root_input_dir should look like:
+
+root_input_dir/ 
+ - sch-haematobium/
+   - PopulationMetadatafile.csv
+   - **/ntdmc-*.csv
+ - sch-mansoni-high-burden/
+   - PopulationMetadatafile.csv
+   - **/ntdmc-*.csv
+ - sch-mansoni-low-burden/
+   - PopulationMetadatafile.csv
+   - **/ntdmc-*.csv
+
+## skip_canonical
+
+In all cases, there is an optional parameter `skip_canonical`, if this is
+set to `True` then the canonical step will be skipped. 
+
+This step is required, if you skip it you will get the following error:
```

Comment:
:bulb: For clarity, I might say `The canonical step is required if the canonical data has not been generated previously`